src/qrc_importer.py

```diff
@@ -16,11 +16,33 @@
 #
 
 import sys
+from importlib import abc
+from importlib.util import spec_from_loader
+
 import pyotherside
 
-from importlib import abc
+
+class PyOtherSideQtRCLoader(abc.Loader):
+    def __init__(self, filepath):
+        self.filepath = filepath
+
+    def create_module(self, spec):
+        return
```

Comment:
Make it more explicit that this returns `None` as per [the docs](https://docs.python.org/3/library/importlib.html#importlib.abc.Loader.create_module):

```suggestion
        return None
```